src/components/FindWallet/WalletTable/WalletMoreInfo.tsx

```diff
@@ -48,7 +72,7 @@ export const WalletMoreInfo = ({
 
   return (
     <Box>
-      <SimpleGrid templateColumns="65px auto">
+      <SimpleGrid columns={1} templateColumns="65px auto">
```

Comment:
Is `templateColumns` necessary now that we have just 1 column?
```suggestion
      <SimpleGrid columns={1}>
```

Reply:
```suggestion
      <SimpleGrid templateColumns="65px">
```
?

Reply:
`templateColumns` still needed for the MoreInfo section